From dde82a2ffe63af3a93d0be007d60244119360bb3 Mon Sep 17 00:00:00 2001
From: Tim Lunn <tim@feathertop.org>
Date: Tue, 16 Jan 2024 12:26:26 +1100
Subject: [PATCH] dts fixes from dtbs_check

fix usb warning

/usr/local/src/iHost/ha-operating-system/output/build/linux-6.6.11/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dtb: usb2phy@fe8a0000: otg-port: 'vbus-supply' does not match any of the regexes: 'pinctrl-[0-9]+'
	From schema: /usr/local/src/iHost/ha-operating-system/output/build/linux-6.6.11/Documentation/devicetree/bindings/phy/rockchip,inno-usb2phy.yaml

fix combphy

/usr/local/src/iHost/ha-operating-system/output/build/linux-6.6.11/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dtb: phy@fe830000: 'phy-supply' does not match any of the regexes: 'pinctrl-[0-9]+'
	From schema: /usr/local/src/iHost/ha-operating-system/output/build/linux-6.6.11/Documentation/devicetree/bindings/phy/phy-rockchip-naneng-combphy.yaml
---
 arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dts b/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dts
index 65ba3e99ec62..5e85fc7539a3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-odroid-m1s.dts
@@ -182,7 +182,6 @@ vcc5v0_usb_host: vcc5v0-usb3-regulator {
 };
 
 &combphy1 {
-	phy-supply = <&vcc5v0_usb_host>;
 	status = "okay";
 };
 
@@ -686,7 +685,7 @@ &usb2phy0_host {
 };
 
 &usb2phy0_otg {
-	vbus-supply = <&vcc5v0_usb_otg>;
+	phy-supply = <&vcc5v0_usb_otg>;
 	status = "okay";
 };
 
-- 
2.40.1

